Q/QuantumEspresso/build_tarballs.jl

```diff
@@ -0,0 +1,45 @@
+using BinaryBuilder, Pkg
+
+name = "QuantumEspresso"
+version = v"6.7.0"
+
+sources = [
+    ArchiveSource("https://github.com/QEF/q-e/releases/download/qe-6.7.0/qe-6.7-ReleasePack.tgz",
+                  "8f06ea31ae52ad54e900a2f51afd5c70f78096d9dcf39c86c2b17dccb1ec9c87"),
+    DirectorySource("bundled"),
+]
+
+
+# Bash recipe for building across all platforms
+script = raw"""
+    cd qe-*
+    atomic_patch -p1 ../patches/0000-pass-host-to-configure.patch
+
```

Comment:
```suggestion
    update_configure_scripts
```

Reply:
Not quite, but good call ... that needs to go into the patch, I think.